src/data-browser/maps/MapsGraphs.jsx

```diff
@@ -20,18 +19,26 @@ const MapsGraphs = props => {
           <p className='lead'>
             The HMDA Maps tool allows you to explore subsets of HMDA data,
             filtered by{' '}
-            <PopularVariableLink year={year}>
+            <a
+              href={
+                '/documentation/tools/data-browser/data-browser-filters#action-taken-action_taken'
+              }
```

Comment:
Consider linking to the section header just to help orient users post-click.
```suggestion
          href='/documentation/tools/data-browser/data-browser-filters#pre-selected-filters'
```

Reply:
URL has been updated!